src/cgame/cg_beacon.cpp

```diff
@@ -148,6 +160,28 @@ else if( !Q_stricmp( token.string, #x ) ) \
 	Parse_FreeSourceHandle( fd  );
 }
 
+/**
+ * @brief Sets location on exact center of target player entity.
```

Comment:
This comment is not true for human beacons, i. e. about half the time.

Reply:
Updated